app/views/course/achievements/_form.html.slim

```diff
@@ -5,6 +5,10 @@
 
   = f.input :description
 
+  = f.label t('.badge')
+  div.achievement-form = display_achievement_badge(@achievement)
+  = f.file_field :badge, class: 'upload-btn'
```

Comment:
Yes, the global form styling can be overridden since Simple Form's markup is consistent.

OK, standardise after the course logo is done.